.rubocop.yml

```diff
@@ -21,4 +21,4 @@ Metrics/MethodLength:
   Max: 15
 
 Metrics/ClassLength:
-  Max: 150
+  Max: 154
```

Comment:
If it's worth refactoring this class to make it shorter I'd rather do it separately